adam-core/src/test/scala/org/bdgenomics/adam/util/ADAMFunSuite.scala

```diff
@@ -25,6 +25,7 @@ trait ADAMFunSuite extends SparkFunSuite {
   override val properties: Map[String, String] = Map(("spark.serializer", "org.apache.spark.serializer.KryoSerializer"),
     ("spark.kryo.registrator", "org.bdgenomics.adam.serialization.ADAMKryoRegistrator"),
     ("spark.kryoserializer.buffer.mb", "4"),
-    ("spark.kryo.referenceTracking", "true"))
+    ("spark.kryo.referenceTracking", "true"),
+    ("spark.hadoop.parquet.task.side.metadata", "false"))
```

Comment:
Filtering will still work, it's just that row group elimination only occurs on the client side. We could remove this change (leave parquet.task.side.metadata set to true) and the test will still pass.